### PR TITLE
LazyLoad IntersectionObserver update #2037

### LINKED ISSUE #2037: LazyLoad not always loading properly
### What steps will reproduce the problem?
When using the LazyLoad widget, the images are not always loading as expected. They appear when resizing the window, but that's not a good solution.  

### What is the expected result?
Proper image loading.

### What do you get instead?
[Not loading as expected](https://i.imgur.com/3xzM9BY.png)

### PR DESCRIPTION
**Pull request by TheMaaarc**
See Changelog and Issue #2037.

This closes #2037

### CODE REVIEW

File: core/UPGRADE.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ LUYA UPGRADE
 
 This document will help you upgrading from a LUYA Version into another. For more detailed informations about the breaking changes **click the issue detail link**, there you can examples of how to change your code.
 
+## 1.5.1
```
Comment (nadar) [inline core/UPGRADE.md:6]: version 1.6 should be good

File: core/lazyload/IntersectionObserverPolyfillAsset.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace luya\lazyload;
+
+use luya\web\Asset;
+
+/**
+ * The Lazyload asset files provides all required resources and implements jquery.
+ *
+ * @author Basil Suter <basil@nadar.io>
```
Comment (nadar) [inline core/lazyload/IntersectionObserverPolyfillAsset.php:10]: @TheMaaarc if have not created this file :smile:

File: core/lazyload/IntersectionObserverPolyfillAsset.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ * @since 1.0.0
```
Comment (nadar) [inline core/lazyload/IntersectionObserverPolyfillAsset.php:11]: since 1.6.0

File: core/lazyload/LazyLoad.php [hunk 4]
```diff
@@ -7,6 +7,8 @@
 use yii\helpers\Html;
 use luya\web\View;
 
+use function GuzzleHttp\json_encode;
```
Comment (nadar) [inline core/lazyload/LazyLoad.php:10]: I don't think this is required ...?

File: core/lazyload/IntersectionObserverPolyfillAsset.php (hunk 2 above)
Comment (TheMaaarc) [reply core/lazyload/IntersectionObserverPolyfillAsset.php:10]: Ahhh, you just don't know it yet 😄

File: core/lazyload/LazyLoad.php [hunk 5]
```diff
@@ -71,6 +71,18 @@ class LazyLoad extends Widget
      */
     public $options = [];
 
+    /**
+     * @var array Legacy support for older Browsers (Adds the IntersectionOberserver Polyfill, default: true)
+     * @since 1.5.1
```
Comment (nadar) [inline core/lazyload/LazyLoad.php:76]: should be 1.6

File: core/lazyload/LazyLoad.php [hunk 6]
```diff
@@ -71,6 +71,18 @@ class LazyLoad extends Widget
...
+     */
+    public $legacySupport = true;
+
+    /**
+     * @var array Optionally disable the automatic init of the lazyload function so you can override the JS options
+     * @since 1.5.1
```
Comment (nadar) [inline core/lazyload/LazyLoad.php:82]: should be 1.6

**Comment by nadar** [comment]
@TheMaaarc it seems that the node tests are failing https://github.com/luyadev/luya/pull/2038/checks?check_run_id=889017792

**Comment by TheMaaarc** [comment]
@nadar There seems to be an issue with the test. It works like 50% of the time. We can remove that test, that's the only thing I can offer. I don't have time to debug that atm.

**Pull request merged**